source/reference/operator/query-others.txt

```diff
@@ -0,0 +1,37 @@
+=====================
+Other Query Operators
+=====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/extracts/operators-toc-explanation.rst
+
+.. list-table::
+   :widths: 30,70
+   :header-rows: 1
+
+   * - Name
+
+     - Description
+
+   * - :query:`$comment`
+
+     - Allows use of commented code within the query language.
```

Comment:
Why change the $comment definition on your $rand ticket? This is not what $comment does ...